CONTRIBUTOR_ROLES.md

```diff
@@ -99,7 +99,7 @@ Maintainers are first and foremost contributors that have shown they are committ
 
 To become a project Maintainer, you must meet the following requirements:
 
-* You have been a Member for at least 1 month.
+* You have been a Member for at least 1 month. You are also a Triager.
```

Comment:
Does this mean: "You have been a Triager for at least 1 month"? It is not clear what the added sentence means here.

Reply:
I moved this elsewhere and explained there may be a situation when a new maintainer is not a triager member, yet; and what to do then (add to both groups.)